doc/implementation/geometry.rst

```diff
@@ -11,55 +11,75 @@ Detector geometry descriptions for HEP are almost universally defined using a
 hierarchy of fully nested volumes, often saved as a GDML file
 :cite:`gdml-2006`. These volumes can be represented as a directional
 acyclic graph (DAG): the nodes are the geometric elements, and the edges
-are a geometry placement (which is associated with a transformation and
-other metadata). In HEP geometries, child nodes may not overlap each other or
-their enclosing parent [#]_ volume.
+are an instantiation of the volume *below* inside the volume *above*. This
+instantiation is associated with a transformation and
+other metadata. In HEP geometries, child nodes may not overlap each other or
+their enclosing parent volume.
 
-.. [#] The Geant4 terms "daughter" and "mother" correspond to "child volume"
-   and "parent volume" in Celeritas.
+.. table:: Celeritas nomenclature tends toward computer science terminology.
 
-To support multiple geometry applications (including detector descriptions that
-are not Geant4 hierarchies), and to make the code
-backend-agnostic for integrating with physics, Celeritas defines abstract
-geometry concepts indexed as IDs.
+   +------------------+-----------+---------------------------------------------+
+   | Celeritas        | VecGeom   | Geant4                                      |
```

Comment:
VecGeom has its own LogicalVolume class, and I think UnplacedVolume corresponds to G4VSolid, not to LogicalVolume.